src/components/post/PostDropDown.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import "../../styles/PostDropDown.scss";
+import Dropdown from "react-bootstrap/Dropdown";
+import editIcon from "../../images/edit-pencil.svg";
+import deleteIcon from "../../images/del-bin.svg";
+import linkIcon from "../../images/link.svg";
+
+class PostDropDown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
same thing here

Reply:
same reason as above, for menu's onclick

Reply:
Does this component ever need a state though? It's just displaying any information that comes in through the props and then passing the onClick to the parent component.